apps/dog-food/Main.hs

```diff
@@ -156,8 +153,7 @@ markdownInfo =
       "  +underlined+    - " <> Markdown Underline "underlined text",
       "  ~strikethrough~ - " <> Markdown StrikeThrough "strikethrough text" <> " (shown as inverse)",
       "  `code snippet`  - " <> Markdown Snippet "a + b // no *markdown* here",
-      "  !r text!        - " <> red "red text" <> " (red, green, blue, yellow, cyan, magenta)",
-      "  !1 text!        - " <> red "also red text" <> " (1-6)",
+      "  !1 text!        - " <> red "red text" <> " (1-6: red, green, blue, yellow, cyan, magenta)",
```

Comment:
I think it was better with both examples, I actually wanted to propose to explain different options (`!<one letter/word/digit> ...!`) in previous pr.. Up to you